dev/js/wow_wizard.js

```diff
@@ -36,128 +36,68 @@
 
 		// Creates theme colors and getter/setters for them.
 		var THEME = (function() {
-      var colors = {
-        pomegranate: {
-            activeIndicatorBackgroundColor: '#F44A56',
-            activeIndicatorTextColor: '#FFF',
-            questionAndAnswerTextColor: '#69181E',
-            circleColor: '#C23640',
-            outlineColor: '#F44A56',
-            lineColor: '#F44A56',
-            activeButtonBackgroundColor: '#F44A56',
-            passiveButtonBackgroundColor: '#FFF',
-            activeButtonTextColor: '#FFF',
-            passiveButtonTextColor: '#000',
-            buttonTextColor: '#FFF',
-            imageChoiceBorderColor: "#F44A56",
-            imageChoiceCircleBackgroundColor: "#FFB3B8"
-        },
-        blueberry: {
-            activeIndicatorBackgroundColor: '#4068D6',
-            activeIndicatorTextColor: '#FFF',
-            questionAndAnswerTextColor: '#204097',
-            circleColor: '#204ABD',
-            outlineColor: '#4068D6',
-            lineColor: '#4068D6',
-            activeButtonBackgroundColor: '#4068D6',
-            passiveButtonBackgroundColor: '#FFF',
-            activeButtonTextColor: '#FFF',
-            passiveButtonTextColor: '#000',
-            buttonTextColor: '#FFF',
-            imageChoiceBorderColor: "#4068D6",
-            imageChoiceCircleBackgroundColor: "#A2B5EB"
-        },
-        peach: {
```

Comment:
Not a problem at all, it's common to discuss code in pull requests, feel free to edit :)